### DISCUSSION TITLE
Back Button

### DISCUSSION THREAD

**Discussion by Mister-Odd** [TriliumNext/Trilium#4557] [Ideas] [answered]
I'm a total noob and having been using your (amazing) software for about 5 min so far... I've never used a PKM or anything like it before... well a bit of OneNote I guess. I'm a .txt file in hundred folders, note taker (gasp!). I'm trying to change my evil ways. 
I've tried about 7 different PKM, Wiki, Terminal, etc. things today - I am easily most impressed with Trilium! (Not that you asked, but I thought my exp and background might be good for your team to know (and my ADHD won't let me be brief)).

While checking out the main demo page, I clicked a link to another note (thingy?) on accident and I immediately wanted to go back to where I was. 

I looked around and I didn't see a way to go back to where I was - I was looking for a back button (preferably on the toolbar).

I found the (home?) demo note (main page?) I was on before easy enough, in the left (navigation) bar and scrolled back to where I was on that note before. So not a big concern. It would have been nice however, to be taken back to where I was - on that original demo note, in the position I was in on that page, before I accidentally clicked the link. - just by clicking "back" or a <--- arrow.

A forward button might be nice too. IE: if I want to go back to a note - look at some data and then flip forward to where I was again.

I know it's not a web browser - but that's where my brain went! 

Just thought you amazing, super-snazzy, UI dev rockstars, would like to know.

(PS the glow effect, on the popup preview, while hovering over a link... is "wow" cool!)

**Comment by zerebos** [comment, marked as answer]
My addon [Trilium Breadcrumbs](https://github.com/rauenzi/Trilium-Breadcrumbs) include a back/forward button to do exactly that, may be worth a look! (See the video demo on that page)

> I know it's not a web browser - but that's where my brain went!

It actually is! You should be able to use normal back shortcuts like you would in a browser including the mouse buttons and `alt`+`<-`.

**Reply by Mister-Odd** [reply]
Nice work! That is even better than I hoped for!

Your addon is actually a better option than baked in - since many people might run this on a server and access it in a browser that has the <- -> already... Your breadcrumbs are dope though, I'd imagine everyone would want that. 

alt+<- ..learned something new today! Thanks for that tip too.. my "Surface" touchpad doesn't let me do a right & left click at the same :-\ 

Thanks for the reply and the work!

**Comment by 0xbismarck** [comment]
If you right click near the Trilium launch bar buttons, you'll see an option that says "Configure Launchbar". Click that, and it'll bring you to a menu where you can add or delete buttons from the launch bar. There are two hidden buttons there that you can make visible for enabling both back and forward buttons. 

As @rauenzi You can also use Alt + the [Left Key] for navigating back a note, and Alt + the [Right Key] for navigating forward.

**Reply by Mister-Odd** [reply]
Thank you! I did find it and have them on the launch bar :)
